WORKSPACE

```diff
@@ -470,9 +470,9 @@ py_library(
 # TODO(fejta): get this to work
 git_repository(
     name = "io_bazel_rules_appengine",
-    commit = "14d860985c2a764fdb6a6072d5450d8360c4ce5b",
+    commit = "fdbce051adecbb369b15260046f4f23684369efc",
     remote = "https://github.com/bazelbuild/rules_appengine.git",
-    #tag = "0.0.5", # Latest at https://github.com/bazelbuild/rules_appengine/releases.
+    #tag = "0.0.8+but-this-isn't-new-enough", # Latest at https://github.com/bazelbuild/rules_appengine/releases.
```

Comment:
What does this mean?

Reply:
The above commit is after 0.0.8, as 0.0.8 remains broken, but head is fixed. However 0.0.8 is the latest release